test/sampling_tests.cpp

```diff
@@ -398,41 +434,63 @@ TEST(SamplingTests, RandomizedSamplingTopKCuda) {
   auto model = Generators::CreateModel(Generators::GetOrtEnv(), MODEL_PATH "hf-internal-testing/tiny-random-gpt2-fp32");
   int batch_size = 5;
   int k = 5;
-  std::vector<int32_t> input_ids{0, 1, 2, 3, 4};
 
   Generators::Config config;
-  config.model.vocab_size = 32000;  // vocab size of llama
+  int vocab_size = 32000;  // vocab size of llama
```

Comment:
Do we need this huge vocab size? Can we just do a tiny amount to make it easier to see what's happening in a debugger?

Why a separate vocab_size variable? The code seems to mix & match between config.model.vocab_size below